app/assets/javascripts/discourse/routes/discovery.js.es6

```diff
@@ -13,8 +13,21 @@ export default DiscourseRoute.extend(OpenComposer, {
   },
 
   beforeModel(transition) {
-    const user = User;
+    // the new bookmark list is radically different to this topic-based one,
+    // including being able to show links to multiple posts to the same topic
+    // and being based on a different model. better to just redirect
     const url = transition.intent.url;
+    if (
+      this.siteSettings.enable_bookmarks_with_reminders &&
+      url === "/bookmarks"
```

Comment:
With this work in a subfolder? I suspect not.

Reply:
@eviltrout in this same area there is also the following:

```
if (
      (url === "/" || url === "/latest" || url === "/categories") &&
      transition.targetName.indexOf("discovery.top") === -1 &&
      User.currentProp("should_be_redirected_to_top")
    ) {
      User.currentProp("should_be_redirected_to_top", false);
      const period = User.currentProp("redirected_to_top.period") || "all";
      this.replaceWith(`discovery.top${period.capitalize()}`);
    }
```

Would that break on subfolder too? Is the alternative to just use `indexOf`?

Reply:
My mistake - looks like `url` here already has had the subfolder removed (I tested it.)